resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/Client.scala

```diff
@@ -401,13 +402,21 @@ private[spark] class Client(
       driverSubmitSslOptions: SSLOptions,
       sslVolumes: Array[Volume],
       sslVolumeMounts: Array[VolumeMount],
-      sslEnvs: Array[EnvVar]) = {
+      sslArgs: Array[String]) = {
     val containerPorts = buildContainerPorts()
     val probePingHttpGet = new HTTPGetActionBuilder()
       .withScheme(if (driverSubmitSslOptions.enabled) "HTTPS" else "HTTP")
       .withPath("/v1/submissions/ping")
       .withNewPort(SUBMISSION_SERVER_PORT_NAME)
       .build()
+    val args = mutable.Buffer[String]()
+    args ++= Seq(
+      "bin/spark-class", "org.apache.spark.deploy.rest.kubernetes.KubernetesSparkRestServer",
```

Comment:
Yes, that's similarly how Mesos works as well (although there is a bit more path wrangling).